components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/db/SitePermissionsEntity.kt

```diff
@@ -43,6 +43,9 @@ internal data class SitePermissionsEntity(
     @ColumnInfo(name = "autoplay_inaudible")
     var autoplayInaudible: SitePermissions.Status,
 
+    @ColumnInfo(name = "media_key_system_access")
+    var mediaKeySystemAccess: SitePermissions.Status,
+
```

Comment:
Only a drive-by. I leave the actual review to @Amejia481 :)

For this change we will have to bump the database version and write a migration so that the field will be added to already existing installations:
- https://github.com/mozilla-mobile/android-components/blob/master/components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/db/SitePermissionsDatabase.kt#L20
- https://github.com/mozilla-mobile/android-components/blob/master/components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/db/SitePermissionsDatabase.kt#L61
- https://github.com/mozilla-mobile/android-components/blob/master/components/feature/sitepermissions/src/androidTest/java/mozilla/components/feature/sitepermissions/db/OnDeviceSitePermissionsStorageTest.kt#L69

Reply:
👍🏽